cost/flexera/cco/scheduled_reports/scheduled_report.pt

```diff
@@ -95,6 +104,43 @@ parameter "param_billing_centers" do
   default []
 end
 
+parameter "param_percent_change_threshold" do
+  type "number"
+  category "Policy Settings"
+  label "Percent Change Threshold Value"
+  description "The threshold (in percent) for the percent change in cost between time period(s) that will trigger the report. Default of 0 will send the report every time based on the applied policy schedule."
+  min_value 0
+  max_value 100
+  default 0
+end
+
+parameter "param_percent_change_consecutive_threshold" do
+  type "number"
+  category "Policy Settings"
+  label "Percent Change Consecutive Threshold"
+  description "The number of consecutive time period(s) where there was a percent change in cost between time period(s) that will trigger the report. Default of 0 will send the report every time based on the applied policy schedule."
+  min_value 1
+  default 1
+end
+
+parameter "param_percent_change_alert_metric" do
+  type "string"
+  category "Policy Settings"
+  label "Percent Change Alert Metric"
+  description "The metric to use for the percent change alert. Percent Change from Previous Time Period Daily Average is recommended to normalize between Month periods.  This has no affect if Billing Term is set to 'Day'."
+  allowed_values "Percent Change from Previous Time Period Daily Average", "Percent Change from Previous Time Period"
```

Comment:
These should probably be shorted up a bit. The description should provide the additional context.

Reply:
These values match the resulting CSV column names which I thought would help user understand what they are tied to.  Open to suggestions if you have something you think would be better here, easy to change